nixos/modules/services/web-apps/tachidesk-server.md

```diff
@@ -0,0 +1,108 @@
+# Tachidesk-server {#module-services-tachidesk-server}
+
+A free and open source manga reader server that runs extensions built for Tachiyomi.
+
+## Basic usage {#module-services-tachidesk-server-basic-usage}
+
+By default, the module will execute tachidesk-server backend and web UI:
+
+```nix
+{ ... }:
+
+{
+  services.tachidesk-server = {
+    enable = true
+  };
+}
+```
+
+It runs in the systemd service named `tachidesk-server` in the data directory `/var/lib/tachidesk-server`.
+
+You can change the default parameters with some other parameters:
+```nix
+{ ... }:
+
+{
+  services.tachidesk-server = {
+    enable = true;
+
+    dataDir = "/var/lib/tachidesk"; # Default is "/var/lib/tachidesk-server"
+    openFirewall = true;
+
+    config = {
+      port = 4567;
+    };
+  };
+}
+```
+
+If you want to create a desktop icon, you can activate the system tray option:
+
+```nix
+{ ... }:
+
+{
+  services.tachidesk-server = {
+    enable = true;
+
+    dataDir = "/var/lib/tachidesk"; # Default is "/var/lib/tachidesk-server"
+    openFirewall = true;
+
+    settings = {
+      server.port = 4567;
+      enableSystemTray = true;
```

Comment:
I think this one is hidden behind `server` as well. https://github.com/Suwayomi/Suwayomi-Server/wiki/Configuring-Tachidesk-Server